pkg/storage/gcs.go

```diff
@@ -64,7 +66,8 @@ func (gcs *GCS) Connect() error {
 		clientOptions = append([]option.ClientOption{option.WithoutAuthentication()}, clientOptions...)
 		clientOptions = append(clientOptions, option.WithEndpoint(endpoint))
 	} else if gcs.Config.CredentialsJSON != "" {
-		clientOptions = append(clientOptions, option.WithCredentialsJSON([]byte(gcs.Config.CredentialsJSON)))
+		d, _ := base64.StdEncoding.DecodeString(gcs.Config.CredentialsJSON)
```

Comment:
Sorry, these changes will break compatibility with current behavior

Could you implement it as a separate config option?
Like `gcs.Config.CredentialsJSONEncoded`

Reply:
Moreover, could you add tests/integration/integration_test.go test case to cover these changes?

Reply:
@Slach thanks for reviewing, I don't see any integration test for CredentialsJSON, I only see its for "WithCredentialsFile". Could you share the link to it ? Moreover, how can I run integration tests locally ?

Reply:
unfortunately, no one test case for `CredentialsJSON` in `tests/integration/` only in `tests/testflows` in python

look https://github.com/AlexAkulov/clickhouse-backup/search?q=QA_GCS_CRED_JSON for details

Reply:
@Slach I am not bale to test it properly
first of all, I have not related issue
https://github.com/golang/go/issues/54291
secondly, if I run without options "-gcflags "all=-N -l" it runs successfully, then, if I run test
`python3 ./test/testflows/clickhouse_backup/regression.py --only ".test/testflows/clickhouse_backup/tests/cloud_storage.py"`
I got 
`docker.errors.DockerException: Error while fetching server API version: ('Connection aborted.', FileNotFoundError(2, 'No such file or directory'))`
could you helm how I can configure env for testing on MacOS ?

Reply:
did you read 
https://github.com/AlexAkulov/clickhouse-backup/blob/master/test/testflows/README.md ?

Reply:
@Slach 
yes, but it's now working on my pc.